test/test_rpc.ts

```diff
@@ -71,4 +75,50 @@ describe("Rpc", () => {
     });
 
   });
+
+  describe("checker", () => {
+
+    it("should allow calling methods that exist", async () => {
+      const rpc = new Rpc(defaults);
+      rpc.start((msg) => rpc.receiveMessage(msg));
+      rpc.registerImpl<ICalc>("calc", new Calc(), checkersForICalc);
+      const stub = rpc.getStub<ICalc>("calc", checkersForICalc);
+      assert.equal(await stub.add(4, 5), 9);
+    });
+
+    it("should forbid calling methods that do not exist", async () => {
+      const rpc = new Rpc(defaults);
+      rpc.start((msg) => rpc.receiveMessage(msg));
+      rpc.registerImpl<ICalc>("calc", new Calc(), checkersForICalc);
+      const stub = rpc.getStub<ICalc>("calc", checkersForICalc);
+      // "any" cast needed to avoid typescript catching the error
+      assert.throws(() => (stub as any).additionify(4, 5), /is not a function/);
+    });
+
+    it("should catch missing methods at typed stub", async () => {
```

Comment:
This doesn't seem to test anything different than what the previous test case tests, does it?